Automattic/wp-calypso
# Components: Add an illustration guide to /devdocs

#### Changes proposed in this Pull Request

See #42004 for the backstory.

* Adds all illustrations to a new reference page in `/devdocs` to make it easier to see them collectively.
* I wanted to add something that allows you to click an image to copy the line you'd import in a new project, but I haven't figured out how to both import the assets via webpack and keep the private URL intact. Obviously this could be a more valuable visual reference in the future, with sorting and searching and the like; right now, it's just a page of static images. :)
* It's entirely possible some of these aren't being used at the moment and should be removed; likewise, I've probably missed some, since not all illustrations live within `/illustrations`.

<img width="1680" alt="Screen Shot 2020-09-24 at 4 52 14 PM" src="https://user-images.githubusercontent.com/2124984/94199105-4f6f7a00-fe86-11ea-9b96-2b3039a657c6.png">

#### Testing instructions

* Switch to this branch and navigate to `/devdocs/illustrations`; you should see a screen of illustrations like the above!
